### PR TITLE
Update grammar at readme

### PR DESCRIPTION
**Pull request by orenyodfat**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
...
-This is the main general swear contract which conduct the trial.
+This is the main general swear contract which conducts the trial.
 
 It calls the specific [trial rules contract](#trial-rules-contract) to get the [witnesses](#witness-contract),get the trial statuses and transitions
 and to check for expiry time for a specific case and trial status.
 
-It calls the specific game witnesses contracts to testimony and validate for a submitted evidence.
+It calls the specific game witnesses contracts to testimony and validates for a submitted evidence.
 
 The SwearGame contract is aligned with the ABIs defined at SwearGameAbstract (courtroomabstract.sol).
```
Comment (MrTibbles) [inline README.md]: Change SwearGameAbstract to a link to the contract.

File: README.md [hunk 2]
```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
 
 ### SwearGame contract
 
-This is the main general swear contract which conduct the trial.
+This is the main general swear contract which conducts the trial.
```
Comment (MrTibbles) [inline README.md]: You just go straight into terminology that has not been explained. What is "Swear contract"? What trial is being conducted? Sw3 has not been explained yet, so you can not use references to it.
When i said you should have a section that explains the terms and concepts being proposed this is what i meant.

File: README.md [hunk 3]
```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
...
 
 It calls the specific [trial rules contract](#trial-rules-contract) to get the [witnesses](#witness-contract),get the trial statuses and transitions
 and to check for expiry time for a specific case and trial status.
```
Comment (MrTibbles) [inline README.md]: This whole paragraph needs revisiting. It is not clear and **uses references to terms a reader is not aware of**.

File: README.md [hunk 4]
```diff
@@ -8,26 +8,26 @@ The courtroom suite includes an abstract generic [SwearGame contract](#sweargame
...
 
-It calls the specific game witnesses contracts to testimony and validate for a submitted evidence.
+It calls the specific game witnesses contracts to testimony and validates for a submitted evidence.
 
 The SwearGame contract is aligned with the ABIs defined at SwearGameAbstract (courtroomabstract.sol).
 
 /// @notice () - a payable function which is used by the service as a deposit functionality
 
-///The function without name is the default function that is called whenever anyone sends funds to a contract
+///The function without name is the default function that is called when anyone sends funds to a contract
```
Comment (MrTibbles) [inline README.md]: I think you need to do some more reading:
http://solidity.readthedocs.io/en/develop/contracts.html#fallback-function

File: README.md [hunk 5]
```diff
@@ -159,7 +159,7 @@ function expired(bytes32 caseId,uint8 status) returns (bool);
 
 /// @return bool - true if it actually start counting for the grace period
 
-///                false -if the grace period already started
+///                false if the grace period already started
 
 function startGracePeriod(bytes32 caseId,uint8 status) returns (bool);
```
Comment (MrTibbles) [inline README.md]: All of the above has simply been copy+pasted from the abstract contracts, ignoring that its not formatted or anything here. That is not what a README is for. You havent even gone into more detail or explained the reasons for the implementation choices.

File: README.md (hunk 2 above)
Comment (orenyodfat) [inline README.md]: Added a link to Sw3 doc as the concept of the courtroom is based on

File: README.md (hunk 3 above)
Comment (orenyodfat) [inline README.md]: please be more specific re this one...
Please see pr #7

File: README.md (hunk 1 above)
Comment (orenyodfat) [inline README.md]: Done in pr #7

File: README.md (hunk 4 above)
Comment (orenyodfat) [inline README.md]: This function is removed at pr #7

File: README.md (hunk 5 above)
Comment (orenyodfat) [inline README.md]: please take a look on pr #7

**Comment by orenyodfat** [comment]
Decided to close this PR due to pr #7 which includes a readme update.

**Pull request closed without merging**